### DISCUSSION TITLE
MOOSE on Xsede

### DISCUSSION THREAD

**Discussion by mamivand** [idaholab/moose#18997] [Q&A General] [answered]
Hello MOOSE users,
Just wondering if anyone has experience in running MOOSE on Xsede machines? Any recommendation for machine selection? 

Thanks,
Mahmood Mamivand

**Comment by GiudGiud** [comment, marked as answer]
Hello

I have not. Let s see if someone pitches in. 
@friedmud have you heard of anyone?

With regards to machine selection, pick something that has:
- CPUs with lots of cores
- enough memory for your application
- an infiniband network or similar if you are going to use more than one node. You want small latencies between your compute nodes, as there can be quite a bit of data transfers between processes, depending on your application
- support modern gcc/mpi etc so you dont have to deal with older versions

Guillaume

**Reply by friedmud** [reply]
I mean - XSEDE is a program, not a machine.  I've used XSEDE a couple of times in the past for running on the Blue Gene machines at ANL.

@GiudGiud is right... you want something with lots of CPU cores.  GPU won't do anything for you with MOOSE at this point.

Is there a particular machine you're interested in?

**Reply by mamivand** [reply]
Derek and Guillaume, 
Thanks for your response. 
Through XSEDE I can request access to machines that are under XSEDE. I wanted to check if anyone has experience in running MOOSE on any of them. Since MOOSE uses Conda, I think my concern regarding MOOSE compatibility with different clusters is not relevant. I am aiming for Jetstream2 Indiana University . 

Mahmood

**Reply by friedmud** [reply]
@mamivand : you won't want to use Conda on the clusters though.  You will need to compile a custom version of MOOSE using the cluster's compilers and MPI stack.

MOOSE from Conda is just for workstations.

You will want to follow our HPC setup instructions here: https://mooseframework.inl.gov/getting_started/installation/hpc_install_moose.html
